client/src/modules/reports/generate/inventory_file/inventory_file.html

```diff
@@ -0,0 +1,60 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.STOCK.INVENTORY_REPORT</h3>
+      <p class="text-info" translate>REPORT.STOCK.INVENTORY_REPORT_DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          <!-- select depot -->
+          <bh-depot-select
+            depot-uuid="ReportConfigCtrl.depot.uuid"
+            on-select-callback="ReportConfigCtrl.setupDepot(depot)"
+            required="true"
+            validate-trigger="ConfigForm.$submitted">
+            <bh-clear on-clear="ReportConfigCtrl.clear('depot')"></bh-clear>
+          </bh-depot-select>
+
+          <!-- select inventory -->
+          <bh-inventory-select
+            inventory-uuid="ReportConfigCtrl.inventory.uuid"
+            on-select-callback="ReportConfigCtrl.setupInventory(inventory)"
+            required="true"
+            validate-trigger="ConfigForm.$submitted">
+            <bh-clear on-clear="ReportConfigCtrl.clear('inventory')"></bh-clear>
+          </bh-inventory-select>
+
+          <!-- choose date until  -->
+          <bh-date-editor
+            label="FORM.LABELS.UNTIL_PERIOD"
```

Comment:
Why does this say "UNTIL_PERIOD" but allow a user to select a date?  Should it not allow a user to selected a period?